### PR TITLE
740 and 747 - stop celery processes on upgrade and uninstall

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: docs/user-guide/release-notes/2.6.x.rst [hunk 1]
```diff
@@ -2,6 +2,18 @@
 Pulp 2.6 Release Notes
 =========================
 
+Pulp 2.6.1
+==========
+
+Bug Fixes
+---------
+
+This is a minor release which contains a fixes for `multiple issues <https://pulp.plan.io/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=cf_4&op%5Bcf_4%5D=%3D&v%5Bcf_4%5D%5B%5D=2.6.1&f%5B%5D=tracker_id&op%5Btracker_id%5D=%3D&v%5Btracker_id%5D%5B%5D=1&f%5B%5D=&c%5B%5D=project&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=cf_5&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=cf_3&group_by=>`_.
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: s/a fixes/bug fixes/

Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: s/multiple issues/these issues/

File: docs/user-guide/release-notes/2.6.x.rst [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+
+A couple of the bugs were related to upgrades. Celery processes running as `pulp_workers`,
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: Replace the first sentence and some of the second with something more proactive. I'll suggest: "One area of improvement relates to upgrades and uninstalls. Starting with 2.6.1, Celery processes...."

File: docs/user-guide/release-notes/2.6.x.rst [hunk 3]
```diff
@@ -2,6 +2,18 @@
...
+`pulp_celerybeat`, and `pulp_resource_manager` are now stopped on update and removal of pulp-server
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: s/now //

Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: s/of /of the /

Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: escape pulp-server since it's a specific software term

File: docs/user-guide/release-notes/2.6.x.rst [hunk 4]
```diff
@@ -2,6 +2,18 @@
...
+package. Just as before, you must restart all Pulp related services after the upgrade.
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: Consider replacing the last sentence with: "After upgrading, you must restart all Pulp related services."

File: docs/user-guide/release-notes/2.6.x.rst [hunk 5]
```diff
@@ -25,6 +37,11 @@ New Features
 
 .. _RabbitMQ: https://www.rabbitmq.com/
 
+Bug Fixes
+---------
+
+This release has fixes for `multiple bugs <https://pulp.plan.io/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=cf_4&op%5Bcf_4%5D=%3D&v%5Bcf_4%5D%5B%5D=2.6.0&f%5B%5D=tracker_id&op%5Btracker_id%5D=%3D&v%5Btracker_id%5D%5B%5D=1&f%5B%5D=&c%5B%5D=project&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=cf_5&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=cf_3&group_by=>`_.
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: This needs to be moved to its own PR and that needs to be merged into 2.6-testing. Otherwise the 2.6.0 release notes will still point to the wrong bug tracker. The content looks right, just not in this PR.

File: docs/user-guide/release-notes/2.6.x.rst [hunk 6]
```diff
@@ -25,6 +37,11 @@ New Features
 
 .. _RabbitMQ: https://www.rabbitmq.com/
 
+Bug Fixes
+---------
+
+This release has fixes for `these issues <https://pulp.plan.io/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=cf_4&op%5Bcf_4%5D=%3D&v%5Bcf_4%5D%5B%5D=2.6.0&f%5B%5D=tracker_id&op%5Btracker_id%5D=%3D&v%5Btracker_id%5D%5B%5D=1&f%5B%5D=&c%5B%5D=project&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=cf_5&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=cf_3&group_by=>`_.
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: This needs to be moved to its own PR and that needs to be merged into 2.6-testing. Otherwise the 2.6.0 release notes will still point to the wrong bug tracker. The content looks right, just not in this PR.

File: docs/user-guide/release-notes/2.6.x.rst [hunk 7]
```diff
@@ -2,6 +2,18 @@
...
 
+Pulp 2.6.1
+==========
+
+Bug Fixes
+---------
+
+This is a minor release which contains bug fixes for `these issues <https://pulp.plan.io/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=cf_4&op%5Bcf_4%5D=%3D&v%5Bcf_4%5D%5B%5D=2.6.1&f%5B%5D=tracker_id&op%5Btracker_id%5D=%3D&v%5Btracker_id%5D%5B%5D=1&f%5B%5D=&c%5B%5D=project&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=cf_5&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=cf_3&group_by=>`_.
+
+One area of improvement relates to upgrades. Starting with 2.6.1, Celery processes running as
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: s/Celery processes running as/Pulp processes /

File: docs/user-guide/release-notes/2.6.x.rst [hunk 8]
```diff
@@ -2,6 +2,18 @@
...
+`pulp_workers`, `pulp_celerybeat`, and `pulp_resource_manager` are stopped on update and removal of
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.6.x.rst]: s/update and removal/upgrade or removal/

**Comment by dkliban** [comment]
ok test

**Comment by bmbouter** [comment]
I can't see why the PR builder would fail with this, but it needs to pass before its merged. Also both 740 and 747 make sure its clear to QE on how to test it. I didn't read carefully to see if it was clear or not. Regarding the content of the PR, LGTM.

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test
